test/static-resize-bilinear-2d.cc

```diff
@@ -338,6 +379,71 @@ TEST_F(StaticResizeBilinear2DTestQU8, matches_operator_api)
   ASSERT_EQ(subgraph_output, operator_output);
 }
 
+TEST_F(StaticResizeBilinear2DTestF16, matches_operator_api)
+{
+  ASSERT_EQ(xnn_status_success, xnn_initialize(/*allocator=*/nullptr));
+  std::generate(input.begin(), input.end(), [&]() { return fp16_ieee_from_fp32_value(f32dist(rng)); });
+  std::fill(operator_output.begin(), operator_output.end(), fp16_ieee_from_fp32_value(std::nanf("")));
+  std::fill(subgraph_output.begin(), subgraph_output.end(), fp16_ieee_from_fp32_value(std::nanf("")));
+
+  // Call operator API.
+  xnn_operator_t op = nullptr;
+  const xnn_status status = xnn_create_resize_bilinear2d_nhwc_f16(channels, channels, channels, /*flags=*/0, &op);
```

Comment:
Please rebase, the signature of this function has changed. Channel, input_pixel_stride and output_pixel_stride and now passed to reshape as part of the work on reshaping xnnpack subgraphs